winml/test/common/testPch.h

```diff
@@ -3,7 +3,9 @@
 //  Copyright (c) Microsoft Corporation. All rights reserved.
 //
 //-----------------------------------------------------------------------------
+#ifndef _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS
```

Comment:
Just FYI, we can do _SILENCE_blah_DEPRECATION_WARNINGS (such as _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING) to "whitelist" warnings we actually want to ignore, instead of silencing every C++17 deprecation :)

Reply:
Sounds good! I'll keep that in mind :)